canonical/assets.ubuntu.com
# WD-11837 Modify search api to support more filters

## Done

- Modified search API to support more filters
- Modifies the database to include a table for authors
- `author_email` acts as a primary key for authors table and foreign key for assets table.
## QA

- Check out this feature branch
- Run the site using the command `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8017
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Use a get request with following URL (add your custom parameters instead of value) `http://0.0.0.0:8017/manager?tag=value&asset_type=value&product_types=value_1&author_email=value&title=value&start_date=value&end_date=value&sf_campg_id=value&language=value&product_types=value_2`

## Issue / Card

Fixes # [WD-11837](https://warthogs.atlassian.net/browse/WD-11837)

## Screenshots

[if relevant, include a screenshot]


[WD-11837]: https://warthogs.atlassian.net/browse/WD-11837?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
